package.json

```diff
@@ -102,7 +102,7 @@
     "screwdriver-executor-docker": "^4.2.0",
     "screwdriver-executor-k8s": "^13.6.1",
     "screwdriver-executor-k8s-vm": "^3.2.2",
-    "screwdriver-executor-queue": "^2.9.0",
+    "screwdriver-executor-queue": "^3.0.0",
```

Comment:
revert it for now, so that we can get this design merged ? I would assume other PR's taking a bit more time to shape up

Reply:
ok